### PR TITLE
[V2] Replace use-alpha/use-beta with cli-release + custom releases support

### PR DESCRIPTION
**Pull request by H00N24**
## 🚀 Pull Request Overview

This PR replaces `use-alpha` and `use-beta` inputs with `cli-release`, which also supports custom releases for development purposes.

- **Type of Change**: refactor/feature
- **Breaking Changes?**: Replacing `use-alpha` and `use-beta`  with `cli-release` input. 

---

### 📜 Tests Checklist

Before submitting this PR, ensure that all tests pass **and** meet the following conditions:

| Category                  | Status                          | Compilation Comment | Results Review |
|---------------------------|----------------------------------|----------------------|----------------|
| **fail_to_start**         | ❌ Failed with compilation error | 🟢 Yes               | 🔴 No          |
| **violated_rules**        | ✅ Passed, with violations found | 🔴 No                | 🟢 Yes         |
| **verified_rules**        | ✅ Passed without violations     | 🔴 No                | 🟢 Yes         |
| **solana_violated_rules** | ✅ Passed, with violations found | 🔴 No                | 🟢 Yes         |

> Please verify that your changes meet the above conditions for each category of tests.  
> If something doesn't match, investigate before submitting the PR.

---

### 📋 Additional Notes (Optional)

- Any risks, edge cases, or implementation notes for reviewers:

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -25,6 +25,19 @@ inputs:
       ```yaml
       cli-version: 7.0.0
       ```
+  cli-release:
```
Comment (ozcodes) [inline action.yml:28]: do we need to add a default value here ?
